plugins/@grouparoo/braze/__tests__/export/export-records.ts

```diff
@@ -0,0 +1,941 @@
+import path from "path";
+import { helper } from "@grouparoo/spec-helper";
+import { exportBatch } from "../../src/lib/export/exportRecords";
+import { loadAppOptions, updater } from "../utils/nockHelper";
+import { connect } from "../../src/lib/connect";
+import { BrazeClient } from "../../src/lib/client/client";
+import { DestinationSyncModeData } from "@grouparoo/core/dist/models/Destination";
+
+const nockFile = path.join(__dirname, "../", "fixtures", "export-records.js");
+
+// these comments to use nock
+const newNock = false;
+require(nockFile);
+// or these to make it true
+// const newNock = true;
+// helper.recordNock(nockFile, updater);
+
+const appOptions = loadAppOptions(newNock);
+const appId = "ASDA#PLKJHG=87467NRsdfuiy+ijp4aAUA3039==";
+const hugeTime = helper.longTime * 6;
```

Comment:
Another slow one :(

Reply:
Now, it's using the wait until function, so this hugeTime is just in case it takes that long.